Data licensing

Elisa-on-GitHub opened the pull request:
I updated some of the information on this page and made a link to the software licensing page.

Elisa-on-GitHub commented:
I noticed in the preview that this change does not render correctly in the guide. I will investigate.

Jolien-S reviewed (approved):
This looks good, let's include it in the Handbook.

Elisa-on-GitHub commented:
A brief summary of what this PR does:
- It changes the text about data licences on the page Data Licensing
- It adds a reference to the Software Licensing page
- It adds a header to the Software Licensing page and fixes the error with the image not rendering
- It changes licenses to licences where appropriate in both of the abovementioned topics, in accordance with #303 
- It adds headers to the Guide Archiving and Publishing Data to make clearer what the text is about and changed the header 'number' (i.e. number of hashes) to make clearer how the guide is structured

@Jolien-S I am sorry - once I realised that this PR took more work than I'd initially anticipated, I converted it back to draft, but you'd done your review in the meantime. Would you mind reviewing this again?
In particular, would you mind having a look at the guide and its internal structure? I'm a bit uncertain about the situation as screenshotted below:

![image](https://github.com/user-attachments/assets/24350035-f51b-4c5a-a00e-e1a117f6c49a)

The header 'Software licensing' is still part of the Data Licences section, but almost immediately afterwards there is a new header with Software Licensing that gives the actual information. This happens because the guide pulls these sections from their respective topic pages and if we want to change this, we have to change something in the topics. Do you have any suggestions? Remove the software licensing header in the Data Licensing page and refer to software licensing only in one sentence?

Thank you!

Jolien-S commented on guides/publish-and-share.qmd:
````diff
@@ -108,11 +108,12 @@ Instead of archiving research data in a data repository, you may choose to publi
 ```
+### Licensing data and software
 
 ```{.include shift-heading-level-by=2}
````
Doesn't this need to be {.include shift-heading-level-by=3}, because there are three ### (see [Adding a guide](https://rdm.vu.nl/contributing.html#adding-a-guide))?

Jolien-S reviewed (changes requested):
I think I spotted one mistake in one of the include statements, please see the comments.

Suggestion for the problem you described:
* Remove the heading 'software licensing', but keep the paragraph.
* For the reference in this paragraph, add 'or below' and add the link for the subsection in 'Licensing software' in the guide.

Jolien-S commented:
Hi @Elisa-on-GitHub, I now realize that what I suggested above doesn't do the trick. Deleting the heading is still a good idea, I think, but it doesn't make sense to refer to 'below' in the topic data licensing. I think what we could do instead is adding a line in the guide saying something like 'The information on licensing software can also be found below in this guide', and then refer to the relevant subsection in the guide. What do you think?

Elisa-on-GitHub commented:
Hi all,

This PR has followed through with the changes we discussed on 20/02/2024:
- Jolien's requested change was made;
- I changed the file name of topics/data-and-software-licensing.qmd to data-licensing to reflect the split into two pages and also made sure that references to this page elsewhere were also updated;
- I added the sentence in the guide as per Jolien's suggestion.

Can we do a final review and get ready to merge?

chartgerink commented on topics/data-licensing.qmd:
```diff
@@ -0,0 +1,32 @@
+
+In principle, Dataverse allows you to choose your terms of use. If you publish your data in Yoda, there is [guidance](https://yoda.vu.nl/site/getting-started/selecting-license.html) available on how to choose a licence and how to customise licences. Some data repositories require you to use a certain licence if you want to deposit your data with them. At [Dryad](https://datadryad.org/), for examp ...
+
+## Software licensing
+Licences that are suitable for data are usually not suitable for software. Giving your software a licence is very important, because it informs other users how they may use your software and for what you take responsibility. You also usually choose a licence for your software at the very beginning of your software development process. You can read more about software licensing on the [Software Li ...
```
```suggestion
```

My only remaining suggestion is to remove this, as the guide will now have software licensing twice:

<img width="907" alt="Screenshot 2025-02-28 at 16 17 05" src="https://github.com/user-attachments/assets/767f992c-d932-4292-9ab1-1850984c4318" />

Elisa-on-GitHub replied:
Hi Chris,

Yes, but removing that section means that the Data Licensing topic does not refer to the Software Licensing topic at all, and I am not sure if that is something we want?

chartgerink replied:
I don't know 😊 Topics are self-contained, so referring to software licensing from data licensing may not be necessary?

I trust you and the rest of @ubvu/editors opinion on this.

Elisa-on-GitHub replied:
Jolien and I discussed yesterday and we did decide to eliminate this heading. But I added a reference to the page about software licences in the first paragraph. I feel like this resolves the discussion
